docs/products/kafka/howto/avoid-high-resource-usage-on-kafka-startup-plan.rst

```diff
@@ -0,0 +1,48 @@
+Avoid high resource usage on Kafka® ``Startup-2`` plan
```

Comment:
Suggest changing the title to either of these options: 
 - Optimize Kafka® Startup-2 Plan for low resource usage
 - Optimizing Resource Usage for Kafka® Startup-2 Plan